From: Sven Roederer <devel-sven@geroedel.de>
Date: Sat, 17 Apr 2021 17:37:08 +0200
Subject: luci.mk: add URL and MAINTAINER fields to package-definitions

Add the LUCI_URL and LUCI_MAINTAINER variables to pass them to the buildpackage
defines. Give them some sane defaults and allow overwritting by the individual
package.

Signed-off-by: Sven Roederer <devel-sven@geroedel.de>

diff --git a/luci.mk b/luci.mk
index 65bcf15a57339ab35f8a70732fbe0aad353872c5..cb53aa2bbd7715ceb38dd7180f7979b76c10cc55 100644
--- a/luci.mk
+++ b/luci.mk
@@ -12,6 +12,8 @@ LUCI_DEFAULTS:=$(notdir $(wildcard ${CURDIR}/root/etc/uci-defaults/*))
 LUCI_PKGARCH?=$(if $(realpath src/Makefile),,all)
 LUCI_SECTION?=luci
 LUCI_CATEGORY?=LuCI
+LUCI_URL?=https://github.com/openwrt/luci
+LUCI_MAINTAINER?=OpenWrt LuCI community
 
 # Language code titles
 LUCI_LANG.ar=العربية (Arabic)
@@ -130,6 +132,8 @@ define Package/$(PKG_NAME)
   VERSION:=$(if $(PKG_VERSION),$(PKG_VERSION),$(PKG_SRC_VERSION))
   $(if $(LUCI_EXTRA_DEPENDS),EXTRA_DEPENDS:=$(LUCI_EXTRA_DEPENDS))
   $(if $(LUCI_PKGARCH),PKGARCH:=$(LUCI_PKGARCH))
+  URL:=$(LUCI_URL)
+  MAINTAINER:=$(LUCI_MAINTAINER)
 endef
 
 ifneq ($(LUCI_DESCRIPTION),)
